src/database/sqlite/event/migrations/m031_add_roles.py

```diff
@@ -1,3 +1,4 @@
+from database.sqlite.event.event_store import DEFAULT_CUSTOM_EXEC_MODE
```

Comment:
```suggestion
from database.sqlite.event.event_store import DEFAULT_CUSTOM_EXEC_MODE
```

No import from the project in migrations